---
title: CredentialsNotFoundError (v0.42)
description: API reference for qiskit.providers.ibmq.credentials.CredentialsNotFoundError in qiskit v0.42
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.credentials.CredentialsNotFoundError
---

<Admonition title="Warning" type="caution">
  The package `qiskit-ibmq-provider` is being deprecated and its repo is going to be archived soon. Please transition to the new packages. More information in [https://ibm.biz/provider\_migration\_guide](https://ibm.biz/provider_migration_guide)
</Admonition>

<span id="qiskit-providers-ibmq-credentials-credentialsnotfounderror" />

# qiskit.providers.ibmq.credentials.CredentialsNotFoundError

<Class id="qiskit.providers.ibmq.credentials.CredentialsNotFoundError" isDedicatedPage={true} github="https://github.com/qiskit/qiskit-ibmq-provider/tree/stable/0.20/qiskit/providers/ibmq/credentials/exceptions.py" signature="CredentialsNotFoundError(*message)" modifiers="exception">
  Errors raised when the credentials are not found.

  Set the error message.
</Class>

